games-util/lgogdownloader/patches/lgogdownloader-3.12.patchset

```diff
@@ -1,77 +0,0 @@
-From c65d24ff976fc9914b2b92bb31e0e07844180aaf Mon Sep 17 00:00:00 2001
-From: Begasus <begasus@gmail.com>
-Date: Sat, 21 Oct 2023 21:30:55 +0200
-Subject: Fix for: help2man: can't get `--help' info (for manpage)
```

Comment:
Not needed anymore?

Reply:
Yeah, the code in that section is totally different now and the man page works without patching :)

Reply:
Probably fixed upstream then, thanks!